tests/testthat/test_build_coxph.R

```diff
@@ -19,6 +19,7 @@ test_that("build_coxph.fast basic", {
   expect_false("Data Type" %in% colnames(ret))
   ret <- model_df %>% tidy_rowwise(model, type='permutation_importance')
   ret <- model_df %>% tidy_rowwise(model, type='partial_dependence')
+  expect_true(all(c("estimate", "p.value") %in% colnames(ret))) # Make sure that estimate and p.value are joined to the result for hover on Prediction tab.
```

Comment:
Add a test case to check 'Odds Ratio' column availability in case of logit.

Reply:
Added such a test case.